.travis.yml

```diff
@@ -15,6 +10,10 @@ script:
 
 jobs:
   include:
+    - node_js: "10"
+    - node_js: "12"
+    - node_js: "14"
+    - stage: github page
```

Comment:
That line can be removed. I’ve added it before just to let you know where you should put the snippet. Looks like I wasn’t enough explicit :-)

Reply:
No no, you were! I was just not paying attention enough ^^
Hope this one is the right one! 🤞